src/utils/hash/classHash.ts

```diff
@@ -135,13 +135,16 @@ export function formatSpaces(json: string): string {
  * Compute hinted class hash for legacy compiled contract (Cairo 0)
  * @param {LegacyCompiledContract} compiledContract
  * @returns {string} hex-string
- * No example, as this function is not exported to user
- */
-export default function computeHintedClassHash(compiledContract: LegacyCompiledContract) {
+ * @example
+ * ```typescript
+ * const compiledCairo0 = json.parse(fs.readFileSync("./cairo0contract.json").toString("ascii"));
+ * const result=hash.computeHintedClassHash(compiledCairo0);
+ * // result = "0x293eabb06955c0a1e55557014675aa4e7a1fd69896147382b29b2b6b166a2ac"
+ * ``` */
+export function computeHintedClassHash(compiledContract: LegacyCompiledContract): string {
```

Comment:
can we add a test for this one also? Should be simple to do based on the example?

Reply:
Done.